app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+
+  def update_post_counter
```

Comment:
- [ ] Good job creating the Post model file 👏 , however the Post model should have association with Like model, Kindly make sure the association is performed 👍 

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes // This line should be there 👍 
```